node/wrapper-allocator/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "wrapper-allocator"
```

Comment:
Perhaps `stats-allocator` would be a more fitting name? To reflect on what this wrapper is responsible for.

Reply:
`tracking-allocator`?